### ISSUE TITLE
Add the ability to transfer rotational power via P2P

### ISSUE THREAD

**Issue by rlnt** [RuochenFu21/CreateAppliedKinetics#9]
Would love to attach shafts to a kinetic P2P tunnel to transfer it through my network.

**Comment by furuochen-dev** [comment]
I recommend using [Create: Ender Transmission](https://www.curseforge.com/minecraft/mc-mods/create-ender-transmission)

**Comment by rlnt** [comment]
You announced this feature in your CurseForge description and I think it would be very fitting for your add-on. I am not interested in an external mod. 🤔

**Comment by furuochen-dev** [comment]
You are right. I was thinking it's impossible to transmit kinetic energy across dimensions. Which means if this is added, it will cause a inconsistent behaviour of P2P tunnel. But then I realized P2P tunnel won't go across dimensions. I don't really play AE2, can you confirm this behaviour?

**Comment by furuochen-dev** [comment]
It says `Possible ... in the future` but you are right it would be quite fitting.

**Comment by Mari023** [comment]
> P2P tunnel won't go across dimensions

they can, using a [Quantum Network Bridge](https://guide.appliedenergistics.org/1.20.4/items-blocks-machines/quantum_bridge#root)

**Comment by furuochen-dev** [comment]
> > P2P tunnel won't go across dimensions
> 
> they can, using a [Quantum Network Bridge](https://guide.appliedenergistics.org/1.20.4/items-blocks-machines/quantum_bridge#root)

Then I sadly can't really add this feature due the Create mod source code. I recommend using [Create: Ender Transmission](https://www.curseforge.com/minecraft/mc-mods/create-ender-transmission) for the behaviour you wanted

**Comment by rlnt** [comment]
You could restrict the kinetic P2P frequency to be within the same dimension.

**Comment by furuochen-dev** [comment]
I'll have to keep this idea for later, probably won't be added in a short while
